src/commands/server/start.ts

```diff
@@ -416,7 +415,7 @@ export default class Start extends Command {
     } catch (err) {
       const isEmptyDir = await this.isDirEmpty(ctx.directory)
       if (isEmptyDir) {
-        this.error('There are no any logs available for the current installation')
+        this.error(`${err}\nThere are no any logs available for the current installation`)
```

Comment:
Shouldn't we write something about the fact that installation failed? Just empty installation logs couldn't necessary mean installation failed, IMHO.

Reply:
The logic could be the following:
```
if (isEmptyDir) {
     this.error(`${err}\nInstallation failed. No logs available.`)
} else {
     this.error(`${err}\nInstallation failed, check logs in '${ctx.directory}'`)
}
```

Reply:
Actually `this.error(): never`. 
It terminates the program. The logic with `if` isn't needed.

Reply:
```
if (isEmptyDir) {
     this.error(`${err}\nInstallation failed. There are no available logs.`)
}
this.error(`${err}\nInstallation failed, check logs in '${ctx.directory}'`)